calico_kubernetes/calico_kubernetes.py

```diff
@@ -9,13 +9,17 @@
 from subprocess import check_output, CalledProcessError, check_call
 import requests
 import sh
+import logging
 from netaddr import IPAddress, AddrFormatError
+from logutils import configure_logger
 from pycalico import netns
 from pycalico.datastore import IF_PREFIX, DatastoreClient
 from pycalico.util import generate_cali_interface_name, get_host_ips
 from pycalico.ipam import IPAMClient
 from pycalico.datastore_errors import PoolNotFound
 
+logger = logging.getLogger('kubernetes')
```

Comment:
don't name the logger 'kubernetes' - maybe calico_kubernetes, or just use __name__